src/index.ts

```diff
@@ -148,10 +149,13 @@ interface ResolvedColorUsage extends ColorUsage {
   foreground: string
 }
 
-const HEXRegex = /#[0-9a-f]{3,8}\b/gi
-const RGBHSLRegex = /\b(?:rgb|hsl)a?\([\d\s\-,./%]+\)/gi
+const enableDetectColorLangs = ['css', 'scss', 'less', 'html', 'vue', 'svelte', 'md']
+
+export function detectColorUsage(code: string, lang: string): ColorUsage[] {
+  if (!enableDetectColorLangs.includes(lang.toLowerCase())) {
+    return []
+  }
```

Comment:
It think we should filter this only for named colors. Expressions like `let colors = '#155'` in any languages should also get highlighted. And we could probably introduce some options to give the control to users if needded.